lecli/api_utils.py

```diff
@@ -19,6 +19,8 @@
 
 AUTH_SECTION = 'Auth'
 URL_SECTION = 'Url'
+LOGGROUPS_SECTION = 'LogGroups'
+CLI_FAVORITES_SECTION = 'CLI_Favorites'
```

Comment:
Everything else is camelcase/one word. Why the change? Or if you want to use underscore log groups should also have an underscore (prefer that one)

Reply:
Also all others start with only one uppercase letter

Reply:
I'm changing CLI_Favorites to Cli_Favorites. The LogGroups is there to be used with legacy section in config files users already have on their hosts and so cannot be changed here.